sources/testme/js/kTestMe.js

```diff
@@ -270,7 +270,18 @@ var kTestMe = {
 		
 		this.classes[type].subClasses = subTypes;
 		for(var i = 0; i < subTypes.length; i++)
+		{
 			this.registerClass(subTypes[i]);
+			
+			if(subTypes[i].hasOwnProperty("isArray") && subTypes[i].isArray && subTypes[i].hasOwnProperty("arrayType") && subTypes[i].arrayType.hasOwnProperty("properties"))
```

Comment:
I do need to check this since as we discussed the second keyValueArray object (for example) will not have this fields and again we will have an exception in the code causing the objects not to be fully loaded.

As to checking this does not break loading abstract classes, I tested that the KalturaCuePoint objects are loaded properly.

Anything else you can think of that is worth to check?